CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - [#1061](https://github.com/crypto-org-chain/chain-main/pull/1061) Integrate sdk 0.50.
 - [#1068](https://github.com/crypto-org-chain/chain-main/pull/1068) Upgrade ibc-go to `v8.3.2` and remove icaauth module.
 - [#1084](https://github.com/crypto-org-chain/chain-main/pull/1084) Add MsgModuleQuerySafe in allowed messages for ica host param.
+- [#1086](https://github.com/crypto-org-chain/chain-main/pull/1086) Upgrade ibc-go to `v8.4.0`.
```

Comment:
why not 8.5?

Reply:
get some verification [error](https://github.com/crypto-org-chain/chain-main/actions/runs/11320630765/job/31478475913) might need help from @devashishdxt